frontend/src/Component/signup.js

```diff
@@ -7,39 +7,77 @@ function Signup() {
   let [userEmail, setUserEmail] = useState("")
   let [userPassword, setUserPassword] = useState("")
   let [userFullName, setUserFullName] = useState("")
+  let [userUserName, setUserUserName] = useState("")
 
+  const validatEmail = () => {
+    let email = userEmail.trim()
+    if (email.endsWith('@gmail.com') && email.length > 10) {
```

Comment:
use return (email.endsWith('@gmail.com') && email.length > 10)
makes it simpler 
also change this function if written anywhere else